src/bind/Sketch.cxx

```diff
@@ -34,21 +34,37 @@ void Sketch::_register_methods() {
 #undef U
 
 void Sketch::init(String src, String home_dir) {
-    sketch =
-        smce::Sketch{std_str(src),
-                     {.fqbn = "arduino:sam:arduino_due_x",
-                      .legacy_preproc_libs = {smce::SketchConfig::ArduinoLibrary{"MQTT@2.5.0"},
-                                              smce::SketchConfig::ArduinoLibrary{"WiFi@1.2.7"},
-                                              smce::SketchConfig::ArduinoLibrary{"Arduino_OV767X@0.0.2"},
-                                              smce::SketchConfig::ArduinoLibrary{"SD@1.2.4"}},
-                      .plugins = {smce::PluginManifest{
-                          .name = "Smartcar_shield",
-                          .version = "7.0.1",
-                          .uri = "https://github.com/platisd/smartcar_shield/archive/refs/tags/7.0.1.tar.gz",
-                          .patch_uri = "file://" + (std::filesystem::absolute(std_str(home_dir)) /
-                                                    "library_patches" / "smartcar_shield")
-                                                       .generic_string(),
-                          .defaults = smce::PluginManifest::Defaults::arduino}}}};
+    std::vector<std::string> dep = {"ArduinoGraphics"};
```

Comment:
What has this vector done to you for you to copy it it and thus exclude it from being directly initialized where it's used?

Reply:
If I remember correctly, we got a "C1001: Internal compiler error" when using `.depends = {"ArduinoGraphics"}` directly, and this was just a quick fix for that.